interceptors/logging/payload.go

```diff
@@ -148,9 +148,9 @@ type jsonpbObjectMarshaler struct {
 }
 
 func (j *jsonpbObjectMarshaler) marshalJSON() ([]byte, error) {
-	b := &bytes.Buffer{}
-	if err := JsonPbMarshaller.Marshal(b, j.pb); err != nil {
+	b, err := JsonPbMarshaller.Marshal(j.pb)
+	if err != nil {
 		return nil, fmt.Errorf("jsonpb serializer failed: %v", err)
 	}
-	return b.Bytes(), nil
+	return b, nil
 }
```

Comment:
Can we replace uses of this function altogether with `JsonPbMarshaller.Marshal(pb)`?

Reply:
Sure.

Reply:
Shall I remove `jsonpbObjectMarshaler` from line [145](https://github.com/yashrsharma44/go-grpc-middleware/blob/update-proto/interceptors/logging/payload.go#L145)?